pkg/k8s/rule_translate.go

```diff
@@ -221,7 +221,7 @@ func (k RuleTranslator) deleteToCidrFromEndpoint(
 	return toReleasePrefixes, nil
 }
 
-// PreprocessRules translates rules that apply to headless services
+// PreprocessRules translates rules that apply to services without selectors
```

Comment:
> Thanks! I'm not familiar with the terminology here, so I may be wrong. But looking at the code and history, I see that this function would initially apply rules on headless services indeed. It changed with commit https://github.com/cilium/cilium/commit/a270d52ef158371fa2be06db53df39866af0080c. Based on the code, should the comment be:

Sweet. I guess the egress part is implicit, since there is no way to have ingress rules service. I'll update the comment based on your suggestion, thanks!